lib/richtext_append_controller.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+
+class RichTextAppendController {
+  final StreamController<bool> richTextAppendStreamController = StreamController<bool>();
```

Comment:
please expose the dispose function of this controller to let user dispose the stream. WDYT?